util/dockerfiles/README.md

```diff
@@ -70,12 +68,12 @@ Hello, world!
 # Documentation
 
 Chapel's documentation is [available online](https://chapel-lang.org/docs/).
-Documentation for a specific release is also available: [1.16](https://chapel-lang.org/docs/1.16/), [1.15](https://chapel-lang.org/docs/1.15/).
+Documentation for a specific release is also available: [1.18](https://chapel-lang.org/docs/1.18/), [1.17](https://chapel-lang.org/docs/1.17/).
```

Comment:
With our fancy drop-down menu for versions, I don't think we really need this call out anymore.

Reply:
So you vote for completely removing this section?

Reply:
Yes, to be more specific, I vote for removing line 71.

Reply:
The "https://chapel-lang.org/docs" drop down menu has current plus one previous release.  Since we are supporting current plus previous two releases, should we keep line 71 until the drop down menu can support previous two release.

Reply:
I don't think we want the docs/ drop-down menu to support any previous releases by default (today it has no previous releases, just the current and future one).  We already have problems where Google search results are stale pointing to older versions, and from all the sleuthing I've done, the less you point at old releases, the less this will happen.  If every page has a link to an older release, I'm worried that'll water down our "current" documentation.  We used to have every page point to every previous release, and our search results seem to have gotten better since we adopted the current approach (plus, it makes the pull-down much simpler).

If Chapel weren't still changing so rapidly, I'd probably be more open to this, but I think old docs hurt us more than (say) docs from a previous version of C or Python hurt them.

Reply:
Thanks all for the feedback. I've updated README to reference current and one previous release.